src/user/dtos/user-create-input.dto.ts

```diff
@@ -11,4 +11,8 @@ export class CreateUserInput {
   @IsNotEmpty()
   @IsString()
   password: string;
+
+  @IsNotEmpty()
+  @IsString()
```

Comment:
@yashmurty 
`@IsString({each: true})`
this is the correct one I guess ?  🤔

Reply:
Changing it to `@IsArray`.
Thanks for pointing it out.